### ISSUE TITLE
Access Denied to https://kubeupv2.s3.amazonaws.com/kops/1.12.0-alpha.1/linux/amd64/utils.tar.gz.sha1

### ISSUE THREAD

**Issue by rjanovski** [kubernetes/kops#6577]
**1. What `kops` version are you running? The command `kops version`, will display
 this information.**
Version 1.12.0-alpha.1 (git-e29765c)

**2. What Kubernetes version are you running? `kubectl version` will print the
 version if a cluster is running or provide the Kubernetes version specified as
 a `kops` flag.**
  kubernetesVersion: 1.11.7

**3. What cloud provider are you using?**
aws

**4. What commands did you run?  What is the simplest way to reproduce this issue?**
`kops update cluster $NAME
`

**5. What happened after the commands executed?**
```
➜  kops git:(k8s) ✗ kops update cluster $NAME
I0304 13:41:25.663864   20810 builder.go:297] error reading hash file "https://kubeupv2.s3.amazonaws.com/kops/1.12.0-alpha.1/linux/amd64/utils.tar.gz.sha1": unexpected response code "403 Forbidden" for "https://kubeupv2.s3.amazonaws.com/kops/1.12.0-alpha.1/linux/amd64/utils.tar.gz.sha1": <?xml version="1.0" encoding="UTF-8"?>
<Error><Code>AccessDenied</Code><Message>Access Denied</Message><RequestId>8F438832BE37300C</RequestId><HostId>s6Gp4kzTY0/+GbvWxJI32EAXqzj1kOb4WyT0aP6AaWHh1yFlxhfYGWb4HzTlhOzM/M3dh87d79A=</HostId></Error>
```

**6. What did you expect to happen?**
cluster update simulation info printer to terminal

**7. Please provide your cluster manifest. Execute
  `kops get --name my.example.com -o yaml` to display your cluster manifest.
  You may want to remove your cluster name and other sensitive information.**
(this is just an upgrade of existing cluster (created with 1.11.1), using 1.12-alpha to handle multi-instance IG)

**8. Please run the commands with most verbose logging by adding the `-v 10` flag.
  Paste the logs into this report, or in a gist and provide the gist link here.**
```
...
I0304 13:58:13.425374   22828 urls.go:71] Using default base url: "https://kubeupv2.s3.amazonaws.com/kops/1.12.0-alpha.1/"
I0304 13:58:13.425419   22828 context.go:159] Performing HTTP request: GET https://kubeupv2.s3.amazonaws.com/kops/1.12.0-alpha.1/linux/amd64/utils.tar.gz.sha1
I0304 13:58:14.265671   22828 builder.go:297] error reading hash file "https://kubeupv2.s3.amazonaws.com/kops/1.12.0-alpha.1/linux/amd64/utils.tar.gz.sha1": unexpected response code "403 Forbidden" for "https://kubeupv2.s3.amazonaws.com/kops/1.12.0-alpha.1/linux/amd64/utils.tar.gz.sha1": <?xml version="1.0" encoding="UTF-8"?>
<Error><Code>AccessDenied</Code><Message>Access Denied</Message><RequestId>7BD24A371E7EA19B</RequestId><HostId>SjGThwZt1KIpA9IQdG8hygrWznXJXL7bWo97s82RuKa6DX82X5Q2kWGNhTYW/jZbBcTH0cQZuBU=</HostId></Error>

cannot determine hash for "https://kubeupv2.s3.amazonaws.com/kops/1.12.0-alpha.1/linux/amd64/utils.tar.gz" (have you specified a valid file location?)
```

**9. Anything else do we need to know?**
looks like a missing resource in s3, or bad permissions?

**Comment by kciredor** [comment]
Same here.

**Comment by rjanovski** [comment]
hey @justinsb can you maybe fix this one?
It's probably just an s3 file not exposed as public. 
no one can use the master branch util it is fixed

Thanks :)

**Comment by edsonmarquezani** [comment]
+1

**Comment by edsonmarquezani** [comment]
I also noticed that in this new version `kops-configuration.service` is trying to download `utils.tar.gz` from Github releases, which, off course, aren't available yet. That will block the usage of the master as well, maybe "forever" (because it is always ahead of the last release).

Anyway, I managed to use it applying local patchs for the version number (so it will download assets from the previous version, which are already acessible) and removing that github address for `utils.tar.gz`, otherwise the node bootstrap fails.

This way, I could test mixedInstance setup, which were my reason for going with 1.12.
